NetSDK/Services/Client/Classes/SplitManager.cs

```diff
@@ -39,5 +39,27 @@ public List<LightSplit> Splits()
 
             return lightSplits.ToList();
         }
+
+
+        public SplitView Split(string featureName)
+        {
+            if (splitCache == null)
+            {
+                return null;
+            }
+
+            var split = splitCache.GetSplit(featureName);
+
+            var lightSplit = new SplitView()
+                {
+                    name = split.name,
```

Comment:
This has been fixed right now. Here: https://github.com/splitio/.net-client/pull/31
Thanks Adil